### ISSUE TITLE
Add ability to list package contents

### ISSUE THREAD

**Issue by diversys** [aplgithub/haikudepotserver#127]
It's been quite a few times already when I found myself wanting to see files inside a package without having access to Haiku or using a different Haiku flavour which doesn't have curtain package. Would be nice to have access to dependencies too.

**Comment by aplgithub** [comment]
Yes I have had this thought myself.  I'm sure this would be really useful.
